Lib/test/test_sys.py

```diff
@@ -1475,6 +1479,16 @@ def test_gc_head_size(self):
         # but lists are
         self.assertEqual(sys.getsizeof([]), vsize('Pn') + gc_header_size)
 
+    def test_inline_values(self):
+        vsize = test.support.calcvobjsize
+        gc_header_size = self.gc_headsize
+        inline_values_size = 32
+
+        linked_list = None
+        for i in range(28):
```

Comment:
I find that it needs this many iterations to "stabilize". Is there a right way to do this?